### ISSUE TITLE
nan or gradient explosion problem even using FP32

### ISSUE THREAD

**Issue by hellojialee** [NVIDIA/apex#951]
Hi, I'm using Laplace loss for the offset vector regression task. Even if I manually set the loss to zero at each batch, the optimizer.step() still arise inf/nan problem. 

Could anyone give me some help? Thanks!
